envoy/tests/common.py

```diff
@@ -4,19 +4,20 @@
 
 from datadog_checks.dev import get_docker_hostname, get_here
 
+from .legacy.common import FLAVOR
+
 HERE = get_here()
 FIXTURE_DIR = os.path.join(HERE, 'fixtures')
 DOCKER_DIR = os.path.join(HERE, 'docker')
-ENVOY_LEGACY = os.getenv('ENVOY_LEGACY')
 ENVOY_VERSION = os.getenv('ENVOY_VERSION')
+ENVOY_API_FLAVOR = os.getenv('FLAVOR')
 
 HOST = get_docker_hostname()
 PORT = '8001'
 
 URL = 'http://{}:{}'.format(HOST, PORT)
 DEFAULT_INSTANCE = {'openmetrics_endpoint': '{}/stats/prometheus'.format(URL)}
-LEGACY_INSTANCE = {'stats_url': '{}/stats'.format(URL)}
-requires_new_environment = pytest.mark.skipif(ENVOY_LEGACY != 'false', reason='Requires prometheus environment')
+requires_new_environment = pytest.mark.skipif(FLAVOR == 'api_v2', reason='Requires prometheus environment')
```

Comment:
Is this supposed to be:
```suggestion
requires_new_environment = pytest.mark.skipif(ENVOY_API_FLAVOR == 'api_v2', reason='Requires prometheus environment')
```